configs/tof/tof_x4_official_vimeo90k.py

```diff
@@ -56,7 +56,6 @@
         data_prefix=dict(img='BIx4up_direct', gt='GT'),
         ann_file='meta_info_Vid4_GT.txt',
         depth=2,
-        num_input_frames=7,
```

Comment:
`num_input_frame` should not be delete, since tof use `GenerateFrameIndiceswithPadding` in the transforming pipeline.